src/pages/components/content-switcher/style.mdx

```diff
@@ -1,44 +1,87 @@
+<AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink>
+<AnchorLink>Size</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
 ## Color
 
-Content switchers have two main states: `selected` and `unselected`. By default,
-content switcher buttons are unselected with the selected state using a high
-contrast color.
-
-| Element           | Property         | Color token               |
-| ----------------- | ---------------- | ------------------------- |
-| Unselected button | background-color | Transparent               |
-|                   | text color       | `$text-secondary`         |
-|                   | svg              | `$icon-primary`           |
-|                   | border           | `$border-inverse`         |
-| Selected button   | background-color | `$layer-selected-inverse` |
-|                   | text color       | `$text-inverse`           |
-|                   | svg              | `$icon-inverse`           |
-| Divider           | border           | `$border-subtle`          |
+Content switcher includes two default-enabled content states: **selected** and
+**unselected** with the selected state using a high contrast color.
+
+| Type       | Element    | Property         | Color token               |
+| ---------- | ---------- | ---------------- | ------------------------- |
+| Selected   | Container  | background-color | `$layer-selected-inverse` |
+|            | Label text | text color       | `$text-inverse`           |
+|            | Icon       | svg              | `$icon-inverse`           |
+| Unselected | Container  | background-color | transparent               |
+|            | Label text | text-color       | `$text-secondary`         |
+|            | Icon       | svg              | `$icon-primary`           |
+|            | Border     | border           | `$border-inverse`         |
+|            | Divider    | border           | `$border-subtle` \*       |
+
+<Caption>
+  * Denotes a contextual color token that will change values based on the layer
+  it is placed on.
+</Caption>
+
+<div className="image--fixed">
+
+![Selected and unselected states of the content switcher](images/contentswitcher-usage-4.png)
+
+</div>
 
 ### Interactive states
 
-Hover states only apply to `unselected` buttons.
+Content switcher has three interactive states–**hover**, **focus**, and
+**disabled**. Hover states only apply to the unselected state.
+
+| Type       | State    | Element    | Property         | Color token                |
+| ---------- | -------- | ---------- | ---------------- | -------------------------- |
+| Selected   | Focus    | Container  | background-color | `$layer-selected-inverse`  |
+|            |          | Label text | text-color       | `$text-inverse`            |
+|            |          | Icon       | svg              | `$text-inverse`            |
+|            |          | Border     | inner-border     | `$focus-inset`             |
```

Comment:
I think that for selected focus, the content switcher also gets  a $focus border, in addition to the $inset-focus? Like [button](https://carbondesignsystem.com/components/button/style#primary-button).